renaissance-harness/src/main/scala/org/renaissance/harness/MarkdownGenerator.scala

```diff
@@ -374,8 +374,9 @@ operation, and will pass the result of the `isLast` method to some other events.
 
 To make the harness use an external plugin, it needs to be specified on the command line.
 The harness can load multiple plugins, and each must be enabled using the
-`--plugin <class-path>!<class-name>` option. The `<class-path>` is the class path on which
-to look for the plugin class, and `<class-name>` is a fully qualified name of the plugin class.
+`--plugin <class-path>[!<class-name>]` option. The `<class-path>` is the class path on which
+to look for the plugin class, and `<class-name>` is a fully qualified name of the plugin class
+(the class-name can be omitted when specified inside the plugin JAR in a `MANIFEST.MF` property).
```

Comment:
Sorry to nit-pick, can you sneak the name of the property in here, i.e., that it is the `Renaissance-Plugin` property?

Reply:
Done.